forte/integrals/conventional_integrals.cc

```diff
@@ -165,6 +167,75 @@ void ConventionalIntegrals::set_tei(size_t p, size_t q, size_t r, size_t s, doub
         aphys_tei_bb_[index] = value;
 }
 
+void ConventionalIntegrals::set_tei_from_asints(std::shared_ptr<ActiveSpaceIntegrals> as_ints,
+                                                bool alpha1, bool alpha2) {
+    for (size_t p = 0; p < ncmo_; ++p) {
```

Comment:
These loops are OK, but I wonder if it is safe to say `aphys_tei_aa_ = as_ints->tei_aa_vector();`? Basically, if these two integrals have same dimension, I think we should be OK.

Reply:
The dimension is managed in python code, should always be same. I will probably add a check.

Reply:
I need to test this.